lib/python/test/testsuite/Reader/readFiles.py

```diff
@@ -62,6 +62,12 @@ def __init__(self, fileExtension: str,
             raise TypeError("You must set at least one of the"
                             " values(direction or directiontype)")
 
+        self._fileExtension = fileExtension
+        self._directiontype = directiontype
+
+        if directiontype is None:
+            directiontype = "undefined"
```

Comment:
shouldn't this be:
```suggestion
            self._directiontype = "undefined"
```

Reply:
No, not in this particular case. A different name for directiontype is chosen here, since checkData.checkDirection cannot handle None.

Reply:
Okay